index.d.ts

```diff
@@ -51,5 +51,7 @@ export function getWindows(): number[];
 export function getActiveWindow(): number;
 export function getWindowRect(handle: number): Rect;
 export function getWindowTitle(handle: number): string;
+export function focusWindow(handle: number): void;
+export function resizeWindow(handle: number, width: number, height: number): void;
```

Comment:
**Suggestion**: The size parameter could be an object of shape

```js
{
  x: number;
  y: number;
  width: number;
  height: number
}
```

This way we could perform resizing + repositioning (which is a common combination IMHO).

WDYT?

Reply:
Yeah great point, I completely agree!

Reply:
I'm going to use the Rect interface from line 25